config/jobs/kubernetes-sigs/cluster-api/cluster-api-prowjob-gen.yaml

```diff
@@ -50,6 +50,25 @@ prow_ignored:
         to: "1.32"
       - from: "1.32"
         to: "1.33"
+    release-1.10:
+      testImage: "gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250205-f1f3519e6b-1.32"
```

Comment:
this image should be the same as the one for main above (I think the images were auto-bumped recently)

Reply:
Ok, bumped image version and re-ran the job generation script.